nncf/torch/pruning/utils.py

```diff
@@ -105,16 +112,21 @@ def collect_output_shapes(graph: NNCFGraph) -> Dict[NNCFNodeName, List[int]]:
     :return: Dictionary of output dimension shapes. E.g {node_name: (height, width)}
     """
     modules_out_shapes = {}
+
     output_shape_collecting_info = [
-       (NNCF_GENERAL_CONV_MODULES_DICT, slice(2, None)),
-       (NNCF_LINEAR_MODULES_DICT, slice(None)),
+        OutputShapeCollectionInfo(node_types=[v.op_func_name for v in NNCF_GENERAL_CONV_MODULES_DICT],
+                                  shape_slice=slice(2, None)),
+        OutputShapeCollectionInfo(node_types=[v.op_func_name for v in NNCF_LINEAR_MODULES_DICT],
+                                  shape_slice=slice(None)),
+        OutputShapeCollectionInfo(node_types=['matmul', 'bmm'],
```

Comment:
@daniil-lyakhov Does it affect Filter Pruning algorithm?
Does Filter Pruning algorithm calculate FLOPS after setting some meaningful pruning masks, or it only calculates for FP32 model?

Reply:
FLOPS calculation besides statistics prompt is used to calculate filter pruning level by given FLOPS pruning level. This change possibly affects masks configuration for models that have matmul operations with flops pruning targets in config